### ISSUE TITLE
DoDS Cubemaps textures on bruches

### ISSUE THREAD

**Issue by Speciesx** [ata4/bspsrc#162]
After decompiling, some brushes are assigned with cubemaps textures. 

Example map: [dod_vaterlands_lappland.zip](https://github.com/ata4/bspsrc/files/10474499/dod_vaterlands_lappland.zip)

```
	solid
	{
		"id" "266"
		side
		{
			"id" "267"
			"plane" "(-1811 2794 -211.852) (-1811 2794 -299.852) (-2075 2794 -299.852)"
			"smoothing_groups" "0"
			"material" "maps/dod_telepnevo/wood/woodfloor007a_c17_-1939_2748_-290"
			"uaxis" "[1 0 0 424] 0.25"
			"vaxis" "[0 0 -1 225] 0.25"
			"lightmapscale" "16"
		}

```
Tested with bspsrc-1.4.2 and bspsrc-1.4.3

**Comment by rihi** [comment]
This seems to be caused by bspsrc to heavily relying on the bsp file name. In this case the file name doesn't match the map name in the texture name. 

 I should be able fix that pretty easily.
As a workaround you should be able to just rename the file to `dod_telepnevo.bsp`, which should fix it.

**Comment by rihi** [comment]
Sorry for taking so long, but this should now be fixed with version [1.4.4](https://github.com/ata4/bspsrc/releases).
